active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+Most of the time you simply wrap content with a `ScrollingView` to enable a default vertical scrolling experience when the content is too large to fit in the user visible area.
+
+```xml
+<mux:ScrollingView>
+    <mux:ItemsRepeater ItemsSource="{x:Bind Items}" ItemTemplate="{StaticResource MyTemplate}"/>
+</mux:ScrollingView>
+```
+
+# Layout examples
+
+This section focuses on properties that let you define the visual layout of the `ScrollingView`.
+
+## Setting the scrollable content
+
+This example sets an image that will be displayed at its natural size, but only up to a 500x400 viewport will be displayed at a time. The user can scroll to see the rest of the image.
+
+*`ScrollingView` (new control)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400">
+    <Image Source="Assets/LargeEiffelTower.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing control)*
+
+Similar to above.
+
+
+## Vertical layout
+
+The layout of the `ScrollingView`'s content can be vertical, horizontal, both, or neither (for an example of neither, see the later Photo Viewer example). 
+
+By default the `ScrollingView`'s content layout (orientation) is set to Vertical, which means that the user can scroll the content up and down. Horizontal means left and right.
+
+In the following example, the `ContentOrientation` property is left at its default value of Vertical, with an ItemsRepeater as the `ScrollingView`'s Content. The UniformGridLayout for the ItemsRepeater positions the items horizontally in a row until it runs out of space (500px in this example), then positions the next item on the next row. The ItemsRepeater may then be taller than the 400px that the user can see, but the user can then scroll the content vertically. (The ContentOrientation property defaults to Vertical.)
+
+**`ScrollingView` markup (new)**
+
+```xml
+<mux:ScrollingView Width="500" Height="400">
+    <mux:ItemsRepeater ItemsSource="{x:Bind Albums}" ItemTemplate="{StaticResource MyTemplate}">
+        <mux:ItemsRepeater.Layout>
+            <mux:UniformGridLayout RowSpacing="8" ColumnSpacing="8"/>
+        </mux:ItemsRepeater.Layout>
+    </mux:ItemsRepeater>
+</mux:ScrollingView>
+```
+
+**ScrollViewer markup (existing)**
+
+Similar to above.
+
+## Horizontal layout
+
+In this example the content is a StackPanel that's laying out its items horizontally. The `ScrollingView`'s ContentOrientation property is set to Horizontal to allow the user to scroll left and right to see all of the content.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400"
+                  ContentOrientation="Horizontal">
+    <StackPanel Orientation="Horizontal">
+        <Button Width="300" Content="Button1"/>
+        <Button Width="300" Content="Button2"/>
+        <Button Width="300" Content="Button3"/>
+        <Button Width="300" Content="Button4"/>
+        <Button Width="300" Content="Button5"/>
+    </StackPanel>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
```

Comment:
“In this example, if the buttons happened to be 500px tall, will the vertical scrollbar be visible?”
No. The horizontal StackPanel is given an available height of 400px. Then it cuts its children at 400px because its Height is not set. 

“Will the ScrollingView be scrollable vertically?”
Vertically, you can only overpan it with touch.

On the other hand, it you set the StackPanel Height to 500px, the StackPanel is also given an available height of 400px but ends up being 500px tall. The ScrollingView then becomes vertically scrollable.

Reply:
ScrollBarVisibility is a really confusing feature of the existing ScrollViewer, and one of the biggest changes in the new ScrollingView. In the existing, you use the scroll bar visibility to configure the layout behavior; e.g. disabling a horizontal scroll bar causes text to wrap. In the new, you use the new ContentOrientation property to configure the layout behavior.